controllers/products.js

```diff
@@ -12,5 +14,25 @@
 // UPDATE product
 
 // DELETE product
+export const deleteProduct = (req,res) =>{
+
+    try {
+        const productId = req.params.id * 1;
```

Comment:
I believe you want to turn the string into a number, it's better to use parseInt. 
Also I'd is usually a string anyway.